upgrading.adoc

```diff
@@ -118,29 +118,30 @@ https://github.com/bitcoin/bips/blob/master/bip-0065.mediawiki[BIP65]
 https://github.com/bitcoin/bips/blob/master/bip-0112.mediawiki[BIP113]
 (OP_SEQUENCEVERIFY).
 
-He provides an overview of how upgrade mechanisms have evolved
-throughout the years, up until 2017. After that, as of this writing,
-only one upgrade of the consensus rules has been deployed: Taproot,
-which was deployed in a somewhat different, and chaotic, manner.
+Lombrozo also provides an overview of the way upgrade mechanisms have evolved
+throughout the years, up until 2017. After that year and as of the time of writing,
+only one upgrade of the consensus rules has been implemented: Taproot,
+which was deployed in a somewhat different, and rather chaotic, manner. [comm: written like this, the text seems to suggest that there is a logical connection between Taproot and Segwit; I would either not mention Taproot at this stage or add a sentece at the end of this paragraph explaining briefly what Segwit is and why we are going to analyze it in the next sections]
```

Comment:
Now I understand the rationale behind mentioning Taproot at this stage, thank you. If you agree, I would replace the text from "After that..." with this:

"Since then, only one other major upgrade, which we analyze in <<taproot-deployment>>, has been deployed. The long and somewhat chaotic process that led to its activation has helped us gaining further insights on upgrading mechanisms in Bitcoin."

Reply:
helped us gain further
? I'm unsure.

Reply:
You're right, sorry about that